lib/LocalCentroid.js

```diff
@@ -56,6 +56,46 @@ function cosineDistance2(terms, centroidTerms, scale) {
 }
 
 
+function cosineItemsDistance2(inputItems, centroidItems, scale) {
```

Comment:
I guess you define a separate function in order not to mix with the existing one, but I think they can be merged into one. Maybe in a later PR...